src/player/live-latency-chaser.ts

```diff
@@ -57,6 +59,10 @@ class LiveLatencyChaser {
                 this._on_direct_seek(target_time);
             }
         }
+
+        if (paused && !this._media_element.paused) {
+            this._media_element.pause();
```

Comment:
Really the assignment of `HTMLMediaElement.currentTime` triggers playback resume if the video has been paused?

Reply:
not quite sure, just to be on the safe side